sentences/en/lock_HassGetState.yaml

```
@@ -3,34 +3,43 @@ intents:
   HassGetState:
     data:
       - sentences:
-          - "is <name> {lock_states:state} [in <area>]"
+          - "(is|are) <name> {lock_states:state} [in <area>]"
         response: one_yesno
         requires_context:
           domain: lock
         slots:
           domain: lock
 
       - sentences:
-          - "(is|are) [there] any door[s] {lock_states:state} [in <area>]"
-          - "(is|are) [there] any {lock_states:state} door[s] [in <area>]"
+          - "(is|are) [there] any (door|lock|window)[s] {lock_states:state} [in <area>]"
+          - "(is|are) [there] any {lock_states:state} (door|lock|window)[s] [in <area>]"
+          - "do I have any {lock_states:state} (door|lock|window)[s] [in <area>]"
         response: any
         slots:
           domain: lock
 
       - sentences:
-          - "are all [the] door[s] {lock_states:state} [in <area>]"
+          - "(is|are) all [the] (door|lock|window)[s] [securely] {lock_states:state} [in <area>]"
```

There is no grammatical context in which "is" can be used with "all". Unless you can provide an example, please remove it.

```suggestion
          - "are all [the] (door|lock|window)[s] [securely] {lock_states:state} [in <area>]"
```